workers/unity/Packages/com.improbable.gdk.tools/Common.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Improbable.Gdk.Tools
+{
+    static class Common
+    {
+        public static string CoreSdkVersion { get; }
+
+        private const string PackagesDir = "Packages";
+
+        static Common()
+        {
+            try
+            {
+                var versionFile = Path.Combine(GetThisPackagePath(), "core-sdk.version");
+                CoreSdkVersion = File.ReadAllText(versionFile, Encoding.UTF8).Trim();
+            }
+            catch (Exception e)
+            {
+                Debug.LogException(e);
+            }
+        }
+
+        /// <summary>
+        /// Finds the "file:" reference path from the package manifest.
+        /// </summary>
+        public static string GetThisPackagePath()
+        {
+            const string gdkTools = "com.improbable.gdk.tools";
+            var path = GetManifestDependencies()[gdkTools];
```

Comment:
Perhaps handling the error case instead of throwing an indexing error would be better?

Reply:
Whoops, missed this